install.bat

```diff
@@ -0,0 +1,78 @@
+@ECHO off
+
+ECHO "Welcome to the setup for mentorship-backend repo."
+
+ECHO "This installation requires Python version above 3.0. Make sure you have it installed."
+
+SET /P check_repo="Do you want to clone the repository [y/n] : "
+IF %check_repo%=="y" ( ECHO "Cloning the repository"
```

Comment:
This is a minor error that I came across now, everything else is fine. 
It should be
IF %check_repo%==y
without double quotes as it directly compares the input.

Reply:
Done with the changes.